### ISSUE TITLE
Crash on deleting a workspace

### ISSUE THREAD

**Issue by skolsuper** [terraform-providers/terraform-provider-tfe#176]
## Steps to Reproduce:

1. Create a new workspace and apply:
```hcl
resource "tfe_workspace" "foo" {
  organization          = tfe_organization.main.name
  name                  = "foo"
  operations            = false # set local execution
  queue_all_runs        = false
  file_triggers_enabled = false
}
```

1. Manually delete it in the app.terraform.io UI
1. Also remove it in the terraform config
1. Run `terraform plan` or `terraform apply`

## Expected Result
Nothing will be planned, the resource is gone from both config and remote

## Actual Result
State refresh fails with these errors:
```
Error: rpc error: code = Unavailable desc = transport is closing

Error: rpc error: code = Canceled desc = context canceled
```

(there are a lot of both, but we have a lot of workspaces)

## Versions
```
% tf version
Terraform v0.12.24
+ provider.tfe v0.15.1
```

**Comment by leetrout** [comment]
Thanks @skolsuper - this should be fixed in [v0.16.1](https://github.com/terraform-providers/terraform-provider-tfe/blob/v0.16.1/CHANGELOG.md). Could you upgrade your provider and let us know if the problem still persists?

**Comment by skolsuper** [comment]
> Thanks @skolsuper - this should be fixed in [v0.16.1](https://github.com/terraform-providers/terraform-provider-tfe/blob/v0.16.1/CHANGELOG.md). Could you upgrade your provider and let us know if the problem still persists?

Oh my bad, sorry for the duplicate report.  Thanks!
